properties/size/border-radius.json

```diff
@@ -0,0 +1,12 @@
+{
+  "size": {
+    "border-radius": {
+      "circle": { "value": "50%" },
+      "xs": { "value": "0.25" },
+      "sm": { "value": "0.5" },
+      "md": { "value": ".75" },
+      "lg": { "value": "1" },
+      "xl": { "value": "1.25" }
```

Comment:
Might we be giving consumers too many liberties in defining the look and feel for their particular app? 

I would expect there to be this many options in a package like Bootstrap, because they need to try and accommodate an infinite amount of consumers, each with their own style/tastes. However, in Palmetto's case, we have ~5 apps, and we want to make sure they have the same general look and feel. 

So, in the case of border-radius, might we be better off limiting the options to: flat(0px), 8px, and "circle"?

I suppose the same concern would also apply to colors. We currently have a lot of options for colors, and I'm wondering if the palette should be more constricted, until we decide it needs to be more nuanced. 

Perhaps there is a way, using colors as an example, to maintain the full/nuanced palette in Figma, but only release a subset to be used by the apps. Then, if we determine more options are needed, release the new colors to the apps in a non-breaking way?

Reply:
Good question. I'm trying to anticipate other components that would have a radius, and possible values for those.

For example, eventually we want to add a `size` prop to `<Button />`. For both a smaller and larger sized button, the border-radius is going to scale up/down respectively, not stay 8px.

Other components that will probably make use of these are Cards, Tags and/or Badges (like trello labels), form inputs with different sizes.

Having said that, I can probably remove that last size `xl` as i don't know of an obvious use case for that.